frontend/src/views/Camps.vue

```diff
@@ -113,35 +111,57 @@ export default {
     ContentCard,
     ButtonAdd,
   },
+  data: function () {
+    return {
+      loading: true,
+    }
+  },
   computed: {
     camps() {
       return this.api.get().camps()
     },
     prototypeCamps() {
       return this.camps.items.filter((c) => c.isPrototype)
     },
+    nonPrototypeCamps() {
+      return this.camps.items.filter((c) => !c.isPrototype)
+    },
     upcomingCamps() {
-      return this.camps.items
-        .filter((c) => !c.isPrototype)
-        .filter((c) => c.periods().items.some((p) => new Date(p.end) > new Date()))
+      return this.nonPrototypeCamps.filter((c) =>
+        c.periods().items.some((p) => new Date(p.end) > new Date())
+      )
     },
     pastCamps() {
-      return this.camps.items
-        .filter((c) => !c.isPrototype)
-        .filter((c) => !c.periods().items.some((p) => new Date(p.end) > new Date()))
+      return this.nonPrototypeCamps.filter(
+        (c) => !c.periods().items.some((p) => new Date(p.end) > new Date())
+      )
     },
```

Comment:
I think it would be sensible to include this information in the camps api response. We do not display any real data from the periods. Also in the future I think it would not make sense to load past camps in the gui if not explicitly requested by the user (e.g. opening past camps expansion panel)

Reply:
@usu what do you think about this?